docker-compose-dev.yml

```diff
@@ -164,24 +163,24 @@ services:
     tmpfs:
       - /mail:mode=770,size=128M,uid=1000,gid=1000
 
-  dev_keycloak:
-    container_name: "dev_keycloak"
-    image: 'quay.io/keycloak/keycloak:21.0'
+  keycloak:
+    container_name: keycloak
+    image: keycloak/keycloak:25.0
     hostname: keycloak
+    command:
+      - 'start-dev'
+      - '--http-port=9080'
+      - '--import-realm'
+    volumes:
+      - ./keycloak:/opt/keycloak/data/import
     environment:
-      - KEYCLOAK_ADMIN=kcadmin
-      - KEYCLOAK_ADMIN_PASSWORD=kcpassword
-      - KEYCLOAK_LOGLEVEL=DEBUG
-      - KC_HOSTNAME_STRICT=false
-    networks:
-      dataverse:
-        aliases:
-          - keycloak.mydomain.com #create a DNS alias within the network (add the same alias to your /etc/hosts to get a working OIDC flow)
-    command: start-dev --import-realm --http-port=8090  # change port to 8090, so within the network and external the same port is used
+      KC_HTTP_PORT: "9080"
+      KEYCLOAK_ADMIN: admin@kuleuven.be
```

Comment:
I'm not sure what email to use here. dataverse@localhost is used elsewhere in the file. 🤷